rosdep/python.yaml

```diff
@@ -356,6 +356,33 @@ python-argparse:
       packages: []
     zesty:
       packages: []
+python-attrs:
+  debian: [python-attr]
+  fedora: [python2-attrs]
+  gentoo: [dev-python/attrs]
+  ubuntu:
```

Comment:
this can be simplified to use the wildcard syntax this way we won't have to add manually the rule for future distributions (e.g. cosmic).
```
ubuntu:
  '*': [python-attr]
  trusty:
    pip:
      packages: [attrs]
```

Reply:
done